[2021-12-08 18:49:02,102] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: first_dag.addSafeSearchRows 2021-11-21T18:11:09.696138+00:00 [queued]>
[2021-12-08 18:49:02,122] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: first_dag.addSafeSearchRows 2021-11-21T18:11:09.696138+00:00 [queued]>
[2021-12-08 18:49:02,122] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-12-08 18:49:02,122] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2021-12-08 18:49:02,122] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-12-08 18:49:02,135] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): addSafeSearchRows> on 2021-11-21T18:11:09.696138+00:00
[2021-12-08 18:49:02,141] {standard_task_runner.py:52} INFO - Started process 56 to run task
[2021-12-08 18:49:02,146] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'first_dag', 'addSafeSearchRows', '2021-11-21T18:11:09.696138+00:00', '--job-id', '38', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmply0j7bq0', '--error-file', '/tmp/tmp7i3idy14']
[2021-12-08 18:49:02,147] {standard_task_runner.py:77} INFO - Job 38: Subtask addSafeSearchRows
[2021-12-08 18:49:02,188] {logging_mixin.py:104} INFO - Running <TaskInstance: first_dag.addSafeSearchRows 2021-11-21T18:11:09.696138+00:00 [running]> on host 8c41fc747080
[2021-12-08 18:49:02,225] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=first_dag
AIRFLOW_CTX_TASK_ID=addSafeSearchRows
AIRFLOW_CTX_EXECUTION_DATE=2021-11-21T18:11:09.696138+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-11-21T18:11:09.696138+00:00
[2021-12-08 18:50:44,742] {python.py:118} INFO - Done. Returned value was: None
[2021-12-08 18:50:44,762] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=addSafeSearchRows, execution_date=20211121T181109, start_date=20211208T184902, end_date=20211208T185044
[2021-12-08 18:50:44,794] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-12-08 18:50:44,854] {local_task_job.py:146} INFO - Task exited with return code 0
